requirements/test-django111.txt

```diff
@@ -0,0 +1 @@
+django>=1.11b1,<1.12
```

Comment:
I know that 1.11 should be the last version in 1.x series and next major version planned is 2.0 and not 1.12. Still I think that is better when someone want to restrict package version in `requirements.txt` to `x.y.*` then is better to describe it as `>=x.y,<x.(y+1)` instead of `>=x.y,<(x+1).0` no matter what is the next planned version of that package.

If you still insist that I need to change `1.12` to `2.0` I will change it.